sdk/scala-sdk-protobuf/src/main/scala/kalix/scalasdk/eventsourcedentity/EventSourcedEntityProvider.scala

```diff
@@ -30,7 +29,7 @@ trait EventSourcedEntityProvider[S, E <: EventSourcedEntity[S]] {
 
   def serviceDescriptor: Descriptors.ServiceDescriptor
 
-  def entityType: String
+  def typeId: String
```

Comment:
Is it really a problem to ask users to clean the repo after bumping? TBH, I can't imagine working with proto sdk without cleaning everything before compiling (relatively often). Otherwise, you can face many issues even now. 

In my first implementation, there was a deprecated `entityType` method, but this approach generates a lot of warnings, which can be fixed only by us, by releasing the next SDK version and a clean + compile will be required anyway to get rid of them.

Reply:
No, I had bad luck when thinking, provider is never extended by users themselves, expecting a clean+compile is fine 👍

Reply:
Yeah, I asked @ceecer1 about this, I would leave it as it is because it shouldn't be a big issue to inform the client about the changes. We can revert this decision, but I would like to push this PR forward.